### DISCUSSION TITLE
Durable sessions not working using helm chart and open source version

### DISCUSSION THREAD

**Discussion by GuillermoGarces** [emqx/emqx#13784] [Q&A] [answered]
Hi,

I am using helm chart [chart](https://github.com/emqx/emqx/blob/master/deploy/charts/emqx) to install the open source version of EMQX and I am not being able to enable durable sessions.

I am setting these chart values 

      replicaCount: "1"
      image:
        tag: "5.8.0"
      recreatePods: true
      persistence:
        enabled: true
        size: "10Gi"
        storageClassName: "my-cloud-storage-class"
        accessMode: ReadWriteOnce
      emqxConfig:
        EMQX_DASHBOARD__DEFAULT_USERNAME: "MyAdmin"
        EMQX_DASHBOARD__DEFAULT_PASSWORD: "MyAdminPass"
        EMQX_MQTT__MAX_MQUEUE_LEN: "infinity"
        EMQX_DURABLE_SESSIONS__ENABLE: "true"
        EMQX_CLUSTER__DISCOVERY_STRATEGY: "singleton"

I can see in EMQX dashboard that durable sessions are enabled
Sessions are retained when disconnected, but they are lost if a reset the emqx node (pod).

Am I missing any configuration step?

**Comment by Rory-Z** [comment]
The 5.8.0 is a new version, so I don't know it so much.
@zmstone @id Where are “durable sessions” stored ? Is it still store in EMQX data path ?

**Comment by Rory-Z** [comment, marked as answer]
@GuillermoGarces Sorry, it's a Helm chart mistake, I have fix it in this: https://github.com/emqx/emqx/pull/13791

Could you please add `EMQX_HOST: emqx-0.emqx-headless.default.svc.cluster.local` in `emqxConfig` of `values.yaml` and retry

You may need change `emqx-0` / `emqx-headless` / `default` to your deploymented resource name and your namespace

**Comment by GuillermoGarces** [comment]
Hi @Rory-Z 
Thanks! Adding the EMQX_HOST variable fixed the issue
